### PR TITLE
C350402: Verify that an Order is exported to a definite Vendors Account specified in one of several Integration configurations

### PR DESCRIPTION
**Pull request by IhorBohdan**
Task - [FAT-4562](https://issues.folio.org/browse/FAT-4562)
Test Case - [C368044](https://foliotest.testrail.io/index.php?/cases/view/368044)
[Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1145/)

### CODE REVIEW

**Comment by penfriend** [comment]
please rename the title of the pull request
the main aim of this pull request is the creation of  autotest C368044

File: cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+import ItemRecordView from '../../support/fragments/inventory/itemRecordView';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import NewLocation from '../../support/fragments/settings/tenant/locations/newLocation';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import ItemActions from '../../support/fragments/inventory/inventoryItem/itemActions';
+import ItemRecordEdit from '../../support/fragments/inventory/itemRecordEdit';
+import SwitchServicePoint from '../../support/fragments/servicePoint/switchServicePoint';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import Checkout from '../../support/fragments/checkout/checkout';
+// import Users from '../../support/fragments/users/users';
```
Comment (TetianaParanich) [inline cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js:25]: Do you need this comment?

Comment (IhorBohdan) [reply cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js:25]: Yes, in "after" I describe my issue

File: cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    Orders.selectFromResultsList();
+    Orders.unOpenOrder(orderNumber);
+    OrderLines.selectPOLInOrder();
+    OrderLines.deleteOrderLine();
+    // Need to wait until the order is opened before deleting it
+    cy.wait(2000);
+    Orders.deleteOrderApi(order.id);
+
+    Organizations.deleteOrganizationViaApi(organization.id);
+    // Need to find solution to delete all data, becouse now i cant delete location and user
```
Comment (MaksimTerekhov) [inline cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js:175]: mark it with TODO: something

**Review by MaksimTerekhov — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js (hunk 2 above)
Comment (IhorBohdan) [reply cypress/e2e/orders/items-for-receiving-includes-order-closed-statuses.cy.js]: Done

**Pull request merged**